app/views/external_users/claims/supporting_evidence_checklist/_fields.html.haml

```diff
@@ -3,4 +3,4 @@
     %h3.heading-medium
       = t('.supporting_evidence_checklist')
 
-  = render partial: 'external_users/claims/supporting_evidence_checklist/evidence_checklist', locals: { f: f, collection: DocType.all }
\ No newline at end of file
+  = render partial: 'external_users/claims/supporting_evidence_checklist/evidence_checklist', locals: { f: f, collection: f.object.eligible_document_types }
```

Comment:
This is the point where the `@doc_types` defined in the controller should be used. As you can see by the existent code, it was using `DocType.all` directly in the view, so that variable was not being used at all.